disnake/member.py

```diff
@@ -816,6 +836,19 @@ async def edit(
 
             .. versionadded:: 2.3
 
+        flags: :class:`MemberFlags`
+            The member's new flags. To know what flags are editable,
+            see :ddocs:`the documentation <resources/guild#guild-member-object-guild-member-flags>`.
+
+            If parameter ``verified`` is provided, that will override the setting of :attr:`MemberFlags.bypasses_verification`.
```

Comment:
```suggestion
            If parameter ``bypasses_verification`` is provided, that will override the setting of :attr:`MemberFlags.bypasses_verification`.
```
The original comment coincided with another change, I should've suggested it like this from the start, my bad.

Reply:
ah I forgot that